examples/typescript/simple_sponsored_transaction.ts

```diff
@@ -70,18 +70,8 @@ const example = async () => {
     },
   });
 
-  // Alice signs
-  const senderSignature = aptos.transaction.sign({ signer: alice, transaction });
-
-  // Sponsor signs
-  const sponsorSignature = aptos.transaction.signAsFeePayer({ signer: sponsor, transaction });
-
   // Submit the transaction to chain
-  const committedTxn = await aptos.transaction.submit.simple({
-    transaction,
-    senderAuthenticator: senderSignature,
-    feePayerAuthenticator: sponsorSignature,
-  });
+  const committedTxn = await aptos.signAndSubmitTransaction({ signer: alice, feePayer: sponsor, transaction });
```

Comment:
The purpose of this example was to show you should sign with 2 separate users, rather than have them colocated on the same machine.

Is there another example that shows this?

Reply:
updated it to better represent that they not supposed to be colocated.